doc/cephfs/index.rst

```diff
@@ -1,39 +1,48 @@
 .. _ceph-filesystem:
 
 =================
- Ceph Filesystem
+ Ceph File System
 =================
 
-The Ceph Filesystem (CephFS) is a POSIX-compliant filesystem that uses
-a Ceph Storage Cluster to store its data. The Ceph filesystem uses the same Ceph
-Storage Cluster system as Ceph Block Devices, Ceph Object Storage with its S3
-and Swift APIs, or native bindings (librados).
+The Ceph File System, or **CephFS**, is a POSIX-compliant file system built on
+top of Ceph's distributed object store, **RADOS**. CephFS endeavors to provide
+a state-of-the-art, multi-use, highly available, and performant file store for
+a variety of applications, including traditional use-cases like shared home
+directories, HPC scratch space, and distributed workflow shared storage.
+
+CephFS achieves these goals through the use of some novel architectural
+choices.  Notably, file metadata is separated from file data and served via
```

Comment:
I'd suggest the wording "...file metadata is stored in RADOS, but separated from file data" or something similar. I find that people new to CephFS tend to assume that the MDS cluster requires its own multiple access and HA like Lustre or similar systems.

Reply:
Good suggestion, done!